### ISSUE TITLE
Scrollbar in mobile view navbar not working

### ISSUE THREAD

**Issue by wez3** [wez3/domoboard#64]
The scrollbar in the mobile view navbar is not working correctly on iPhone.

**Comment by Sateetje** [comment]
I bought a Samsung Tab A6 (7") tablet to run Domoboard on the wall (with Volgel's Rong-o).

I have three issues with the Samsung running with Chrome. One of the three is that the scrollbar doesn't scroll.

Is it possible to ignore the mobile view and view the page like on a laptop/pc wth Chrome?

**Comment by squandor** [comment]
just tried an fix for the scroll bug. could you try if its also working on you're situation. At the moment the menu will scroll with the page so you can find all the menu items.

**Comment by miljume** [comment]
I changed to develop branch with `git checkout develop` and then did a `git pull` but i still have issues with the scrolling in Chrome for Android. The left menu scrolls with the page a short bit but you never get to the end of the menu, I only see the first 8 items and I have 13 items in the list (portrait mode). In landscape mode i see 3.5 items

**Comment by Sateetje** [comment]
Scrolling on iPhone Safari is OK. Scrolling on Android Chrome NOK.

**Comment by Sateetje** [comment]
Sorry, looks like it's working. But can't scroll down using the menu itself.

**Comment by squandor** [comment]
thats right its an temporary solution so you can scroll down to the menu using the page itself until we fixed the scrolling in the mobile menu.
